res/client/login.css

```diff
@@ -0,0 +1,4 @@
+QWidget#header
+{
+    background-color: rgba(255,255,255,255);
```

Comment:
An RGBA color value is specified with: rgba(red, green, blue, alpha). The alpha parameter is a number between 0.0 (fully transparent) and 1.0 (fully opaque).